src/data/Assignment.java

```diff
@@ -0,0 +1,42 @@
+package data;
```

Comment:
I think the files in the `src/data` directory are accidental copies of the files in `src/main/java/data`